R/detect_mdd.R

```diff
@@ -0,0 +1,47 @@
+################################################################################
+#
+#' @title Determining Minimum Dietary Diversity (MDD)
+#'
+#' @description Identification of individual 6-23 months old children dietary
+#' practices meet the minimum requirement of dietary diversity (at least 5 food
+#' group out of 8 food groups)
+#'
+#' @param age This parameter holds the information about child age in the month
```

Comment:
Make this clear that this is a numeric vector. R doesn't recognise a "decimal format". This is an ODK concept.